include/vulkan/layer/vk_layer_settings.h

```diff
@@ -1,7 +1,7 @@
 /*
- * Copyright (c) 2015-2023 The Khronos Group Inc.
- * Copyright (c) 2015-2023 Valve Corporation
- * Copyright (c) 2015-2023 LunarG, Inc.
+ * Copyright (c) 2023 The Khronos Group Inc.
+ * Copyright (c) 2023 Valve Corporation
+ * Copyright (c) 2023 LunarG, Inc.
```

Comment:
NOTE: This file is the public interface to the library.

Because it includes files from `Vulkan::Headers` we need to make `Vulkan::Headers` as a dependency.

```
target_link_Libraries(VulkanLayerUtils PUBLIC Vulkan::Headers)
```

We mark it as public because we and the client need it to compile our code.

Reply:
Ok!